grails-forge-core/src/main/java/org/grails/forge/feature/build/gitignore.rocker.raw

```diff
@@ -1,14 +1,37 @@
-Thumbs.db
-.DS_Store
+HELP.md
```

Comment:
I think we can remove this, since it is specific to Spring Applications generated by Spring Initializr

1. HELP.md:
Since Spring Initializr version 2.1.0, new projects include a HELP.md file at the root.
This file provides links to guides and reference documentation relevant to the dependencies you've included in your project.